src/chrome/content/rules/Alpinelinux.org.xml

```diff
@@ -0,0 +1,7 @@
+<ruleset name="Alpinelinux.org">
+	<target host="alpinelinux.org" />
+	<target host="www.alpinelinux.org" />
+
+	<rule from="^http://www\.alpinelinux\.org/" to="https://alpinelinux.org/"/>
```

Comment:
We don't force redirects as long as everything works fine. It's up to the webmasters to do this.